clusterloader2/pkg/measurement/util/pod_store.go

```diff
@@ -40,14 +45,45 @@ type PodStore struct {
 	Reflector *cache.Reflector
 }
 
+var podGroupVersion, _ = schema.ParseGroupVersion("v1")
+var podGropuVersionResource = podGroupVersion.WithResource("pods")
+
+// NewPodStore creates PodStore based on given namespace, label selector and field selector.
+func NewPodStoreForDynamicClient(c dynamic.Interface, namespace string, labelSelector string, fieldSelector string) (*PodStore, error) {
```

Comment:
I don't see any reason for introducing pod store for dynamic client - what was our motivation? It doesn't give us anything.

Reply:
It's because, in my current commit, the caller for func `NewPodStore` does not have clients `clienset.interface`. I will improve it.

Reply:
Done.